addons/website_payment/static/src/js/website_payment_form.js

```diff
@@ -1,7 +1,8 @@
 /** @odoo-module **/
 
-import core, { _t } from "@web/legacy/js/services/core";
+import core from "@web/legacy/js/services/core";
```

Comment:
Why is `core` left in this file?

Reply:
This file still uses the bus exported by core (see line 15/16 in the `start` function)